modules/ROOT/Edits/create-inbound-message-flow.adoc

```diff
@@ -0,0 +1,119 @@
+= Create and Configure an Inbound Message Flow
+:page-aliases: configure-message-flows.adoc
+
+You can create and configure an inbound message flow to receive EDI messages from your trading partners, including messages that go through 3rd party connections. When you create an inbound message flow, you can validate and transform the inbound messages and then send the translated message (JSON or XML) to your backend applications.
+
+To receive messages sent from a trading partner through a 3rd party connection, select an endpoint owned by that connection.
+
+=== Prerequisites
+Before you create your first inbound message flow, you must configure your xref:setup-payload-storage-API.adoc[payload storage API].
+
+=== Steps
+
+. From the Anypoint Partner Manager navigation menu, select *Message Flows*.
+. Click *New message flow > Receive from partner*.
+. Either select an existing partner from the list or select *New* to create a new partner, as described in xref:create-partner.adoc#[Create a partner].
```

Comment:
Do we need this level of specificity given the choices in "Specify the Receiving Endpoint"?